exercises/01-getting-started.md

```diff
@@ -1,63 +1,81 @@
+
+Click `Use this template` on the top of the GitHub page.
 ![Use Template](images/use-template.png)
 
-Fill in the name of your new repository (e. g. *cloud-sdk-team-calendar*) and make sure to create a **private** repository. Click on *Create repository*.
+Fill in the name of your new repository (e. g. _cloud-sdk-team-calendar_) and make sure to create a **private** repository. Click on `Create repository`.
 ![Create New](images/create-new.png)
 
 > ### Why private?
-> We recommend to create your project as a private repository due to licensing constraints. In [exercise 5](05-generate-odata-client.md) we will generate an OData client for SAP SuccessFactors, that is subject to SAP intellectual property and should not be made available as open source. As long as you don't publish these changes, feel free to create a public repository as well.
+>
+> We recommend to create your project as a private repository due to licensing constraints. In [exercise 2](02-generate-odata-clients.md) we will generate OData clients for SAP S/4HANA and SAP SuccessFactors, that is subject to SAP intellectual property and should not be made available as open source. As long as you don't publish these changes, feel free to create a public repository as well.
 
 ## Clone your repository
-On your laptop open a command line and enter the following command. Don't forget to replace your username (and repository name should you have chosen a different one):
+
+On your laptop open a command line and enter the following command.
+
 ```sh
-git clone https://github.com/<your-github-username>/cloud-sdk-team-calendar.git
+git clone https://github.com/YOUR_GITHUB_USERNAME/cloud-sdk-team-calendar.git
 ```
 
 ## Deploy the SAP S/4HANA and SAP SuccessFactors mock server (Optional)
 
->Note: If you have access to real SAP S/4HANA Cloud and SAP SuccessFactors (SFSF) Cloud systems, you may be able to skip this step.
+> Note: If you have access to real SAP S/4HANA Cloud and SAP SuccessFactors Cloud systems, you may be able to skip this step.
 
-Follow these [instructions](https://github.com/SAP/cloud-s4-sdk-book/tree/mock-server#how-to-run-the-server) on how to setup your SAP S/4HANA and SFSF mock server and deploy it on SAP Business Technology Platform (SAP BTP), Cloud Foundry environment.
+Follow these [instructions](https://github.com/SAP/cloud-s4-sdk-book/tree/mock-server#how-to-run-the-server) on how to set up your SAP S/4HANA and SAP SuccessFactors mock server and deploy it on the SAP Business Technology Platform (SAP BTP), Cloud Foundry environment. 
 
-## Setup the destinations environment variable
+**Please checkout the branch [mock-server-update-data](https://github.com/SAP/cloud-s4-sdk-book/tree/mock-server-update-data) for this tutorial instead of the branch [mock-server](https://github.com/SAP/cloud-s4-sdk-book/tree/mock-server) as it is described in the instructions.** The branch has the latest date data as opposed to the branch mock-server.
```

Comment:
Is there a reason why we can't update the `mock-server` branch? Seems like extra effort for the user.

Reply:
date data in the mock-server is 2019. If we use this, a user will have to press go back button for 3 years to show that in this tutorial hence I chose the other branch that has the latest date data.

Reply:
There is a PR open for update the master branch https://github.com/SAP/cloud-s4-sdk-book/pull/50. If this can be merged, we can adjust the instructions. @marikaner

Reply:
thank you for updating the mock-server branch. I removed the instruction for using mock-server-update-data